app/models/admin/edition_filter.rb

```diff
@@ -220,5 +221,9 @@ def location_matches
         " about #{sentence}"
       end
     end
+
+    def only_broken_links
+      true if options[:only_broken_links].present?
```

Comment:
You could do away with the `true if` here. Also, possibly the `.present?`. What happens if it is `.present?` but `false`? (I'm guessing that shouldn't happen). Maybe you could have `!!options[:only_broken_links]` that would always return a boolean but also respect a value of `false`

Reply:
Actually... `.present?` returns `false` if the value is `false` so you could leave it as just `options[:only_broken_links].present?`